project.py

```diff
@@ -81,6 +86,21 @@ def find_board(self, image):
 
         return our_cnt
 
+    def get_seed(self, image):
+        if self.seed is not None:
+            rgb = image[self.seed[1], self.seed[0]]
+            if rgb[1] > rgb[0] and rgb[1] > rgb[2]:
+                return self.seed
+
+        h, w = image.shape[:2]
+        self.seed = (randint(w/4, (w/4)*3), randint(h/4, (h/4)*3))
+        rgb = image[self.seed[1], self.seed[0]]
+        # three retries to find proper seed
+        for i in xrange(1, 3):
+            if rgb[1] > rgb[0] and rgb[1] > rgb[2]:
```

Comment:
@Foxiy If you are really going to use a ghetto green filter then it might make sense to make it a function and then un-ghetto it at some point =)